bin/SimilarityInterface.py

```diff
@@ -0,0 +1,97 @@
+#notes: many functions are incomplete or disabled because sql isn't implemented yet.
+
+import numpy as np #this does math
+import pandas as pd #this helps with the sorting and the math associated
+import argparse #this helps parse the arguments to the call
+import sys #this is needed to deal with inputs
+
+#sql database to look at
+SQLCommand = "null"

```

Comment:
I would suggest making defaults as empty: so for a string this would be `SQLCommand = ''`. This way it still acts as a boolean later on, i.e.
`if SQLCommand:` would return false if unchanged